src/Stratis.Bitcoin/Controllers/Models/BlockModel.cs

```diff
@@ -36,12 +34,10 @@ public class BlockModel
         public object[] Transactions { get; private set; }
 
         [JsonProperty("time")]
-        [JsonConverter(typeof(DateTimeOffsetConverter))]
-        public DateTimeOffset Time { get; private set; }
+        public long Time { get; private set; }
 
         [JsonProperty("mediantime")]
-        [JsonConverter(typeof(DateTimeOffsetConverter))]
```

Comment:
Doesn't `DateTimeOffsetConverter` already convert it to UnixTimeSeconds? Just remove `.ToString()` from converter... I would rather deal with DateTimeOffset in dotnet than with longs

Reply:
@fenix2222 I don't want to make changes to `DateTimeOffsetConverter` as it is used in numerous other places.

Reply:
Hmm, it should be consistent, somewhere we use DateTimeOffset, in other places longs. I think the converter was meant to make it all consistent and probably other usages of it should be fixed. Maybe not for this PR though

Reply:
Whats the issue with longs as opposed to DateTimeOffSet?

Reply:
1. Consistency, all other models mainly use DateTimeOffset, so as a consumer of the model I would expect it to be consistent
2. DateTimeOffset is just a much richer object with helper properties and methods and closer describes the property - Time, CreationTime.